Refactoring preferences

nittka opened the pull request:
This PR primarily addresses #186, but it includes the changes of PR #185.

thSoft commented on org.elysium.parent/org.elysium.ui/src/org/elysium/ui/preferences/LilyPondRefactoringPreferencePage.java:
```diff
@@ -1,20 +1,61 @@
+		dependentFields.add(
+				createBooleanField(REFACTORING_ADAPT_INCLUDES, "Modify includes pointing to refactored files"));
+		dependentFields.add(
+				createBooleanField(REFACTORING_WARN_DELETE_INCLUDED, "Warn if an indluded file is deleted"));
```
Please fix typo: indluded -> included

thSoft commented on org.elysium.parent/org.elysium.ui/src/org/elysium/ui/preferences/LilyPondRefactoringPreferencePage.java:
```diff
@@ -1,20 +1,61 @@
+		dependentFields.add(
+				createBooleanField(REFACTORING_WARN_SEARCHPATH, "Warn if a search path is affected by a refactoring"));
+		dependentFields.add(
+				createBooleanField(REFACTORING_WARN_VARIABLE_INCLUDE, "Warn if a there are variable includes (refactoring not supported)"));
```
Please fix typo: if a -> if

nittka commented:
I fixed the typos and rebased on master.
I assume you still would not like to have the preference for switching off the include modifications. If you changed your mind, adding the additional preference would be easy.

thSoft commented:
Great, thanks!
